posthog/migrations/0126_stricter_team.py

```diff
@@ -0,0 +1,57 @@
+# Generated by Django 3.0.6 on 2021-02-09 09:11
+
+import django.core.validators
+import django.db.models.deletion
+from django.db import migrations, models
+
+import posthog.models.utils
+
+
+def prepare_teams(apps, schema_editor):
+    Team = apps.get_model("posthog", "Team")
+    Team.objects.filter(organization_id__isnull=True).delete()
+    Team.objects.filter(opt_out_capture=True).update(anonymize_ips=True)
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("posthog", "0125_longer_webhook_url"),
+    ]
+
+    operations = [
+        migrations.RunPython(prepare_teams, migrations.RunPython.noop),
+        migrations.AlterField(
+            model_name="team",
+            name="api_token",
+            field=models.CharField(
+                default=posthog.models.utils.generate_random_token,
```

Comment:
Won't this set a random default for all new teams? As in, the default in the db will be a string generated now.

![image](https://user-images.githubusercontent.com/53387/107765560-5e21d500-6d32-11eb-9140-41e5ddf8a674.png)

Reply:
I don't get, it will be random differently for every team